python_scripts/video_pipeline.py

```diff
@@ -0,0 +1,91 @@
+# %% [markdown]
+# # How to define a scikit-learn pipeline and visualize it
+
+# %% [markdown]
+# ### First we load the dataset
+
+# %% [markdown]
+# We need to define our data and target. In this case we will build a classification model
+
+# %%
+import pandas as pd
+
+ames_housing = pd.read_csv("../datasets/house_prices.csv", na_values='?')
+
+target_name = "SalePrice"
+data, target = ames_housing.drop(columns=target_name), ames_housing[target_name]
+target = (target > 200_000).astype(int)
+
+# %% [markdown]
+# We inspect the "head" of the dataframe
```

Comment:
When we put comments, it's better is they bring more understanding than what is already explicit in the function names:

```suggestion
# We inspect the first rows of the dataframe
```